src/thor/dijkstras.cc

```diff
@@ -57,24 +57,30 @@ void Dijkstras::Clear() {
 // Initialize - create adjacency list, edgestatus support, and reserve
 // edgelabels
 template <typename label_container_t>
-void Dijkstras::Initialize(label_container_t& labels, const uint32_t bucket_size) {
+void Dijkstras::Initialize(
+    label_container_t& labels,
+    std::shared_ptr<baldr::DoubleBucketQueue<typename label_container_t::value_type>>& queue,
```

Comment:
the type depends on the method template. i dont think there is a way to do this because a type alias has to happen before the method prototype is declared, which means there is no template to use in the alias.